driver/others/dynamic_power.c

```diff
@@ -36,13 +36,38 @@ char *gotoblas_corename(void) {
 	return corename[0];
 }
 
-#if defined(__clang__)
-static int __builtin_cpu_supports(char* arg) 
+#define CPU_UNKNOWN  0
+#define CPU_POWER5   5
+#define CPU_POWER6   6
+#define CPU_POWER7   7
```

Comment:
POWER7 currently does not exist as an actual, unique target - it is an alias for POWER6. Do you plan to change that ?

Reply:
Fixed